pkg/urbit/vere/pier.c

```diff
@@ -1683,22 +1611,33 @@ u3_pier_discover(u3_pier* pir_u,
   _pier_apply(pir_u);
 }
 
+/* _pier_exit_done(): synchronously shutting down
+*/
+static void
+_pier_exit_done(u3_pier* pir_u)
+{
+  fprintf(stderr, "pier: exit\r\n");
+
+  _pier_work_shutdown(pir_u);
+  _pier_loop_exit(pir_u);
+
+  //  XX uninstall pier from u3K.tab_u, dispose
+
+  //  XX no can do
```

Comment:
What's this about?

Reply:
The function currently shuts down both processes. Obviously not tenable in a multi-tenant environment.